proposals/collection-literals.md

```diff
@@ -151,6 +148,8 @@ _This means an extension method could be added that would silently change how co
 
 Through the use of the [`init`](#init-methods) modifier, existing APIs can directly support collection literals in a manner that allows for no-overhead production of the data the final collection will store.
 
+_Does this support construction of custom dictionary types?_
```

Comment:
TBD at today's meeting.  I think we def want this (i.e. ImmutableDictionary, but we need to find a way to make that possible).